.github/workflows/test.yaml

```diff
@@ -230,8 +230,7 @@ jobs:
       run: |
           # Register a problem matcher to highlight spread failures
           echo "::add-matcher::.github/spread-problem-matcher.json"
-          export CORE_CHANNEL=edge
```

Comment:
why this change? what happens when we leave this in?

Reply:
no I did not get an answer about this over IRC, at least I don't remember it if I did get one :smiley:

Reply:
Sorry for the delay in my response, I removed that because it was wrong, the correct var if we need to set the core channel is SPREAD_CORE_CHANNEL, but in this case is ti not needed because by default is edge.

This is the default which is set in the spread.yaml:
CORE_CHANNEL: '$(HOST: echo "${SPREAD_CORE_CHANNEL:-edge}")'